lte/gateway/python/magma/mobilityd/uplink_gw.py

```diff
@@ -31,6 +33,15 @@ def getIP(self) -> Optional[str]:
             self._current_ip = self._backing_map.get(DHCP_Router_key)
             return self._current_ip
 
+    def read_default_gw(self):
```

Comment:
I feel overall mobilityd structure is growing a lot with these helper files, a good idea could be adding a subdirectory `helpers` or something like that to have these functions there similar to the scripts subdir we 
 already have

Reply:
I can add it as todo, since it is beyond this change.